science/zfp/Portfile

```diff
@@ -44,6 +45,14 @@ variant openmp description {use OpenMP} {
     }
 }
 
-# Note, that tests are likely to fail on 32-bit platforms due to insufficient precision:
-# https://github.com/LLNL/zfp/issues/205
+# See: https://github.com/LLNL/zfp/issues/205
+if {${configure.build_arch} in [list i386 ppc]} {
+    configure.args-append \
+                    -DZFP_INT64='long long' \
+                    -DZFP_INT64_SUFFIX=ll \
+                    -DZFP_UINT64='unsigned long long' \
+                    -DZFP_UINT64_SUFFIX=ull
+}
+
 test.run            yes
+test.cmd-prepend    DYLD_LIBRARY_PATH=${cmake.build_dir}/lib
```

Comment:
You don't want to set t his in `test.env` instead?

Reply:
I think there was some case which led me to generally prefer this form, but I cannot recall now. Here it is perhaps equivalent. (Are these supposed to differ somehow, in theory, or Macports does the same with either?)